test/e2e/suite/serving/cainjector.go

```diff
@@ -324,9 +325,10 @@ var _ = framework.CertManagerDescribe("CA Injector", func() {
 	injectorContext("validating webhook", &injectableTest{
 		makeInjectable: func(namePrefix string) client.Object {
 			someURL := "https://localhost:8675"
+			name := fmt.Sprintf("%s-hook-%s", namePrefix, rand.String(5))
 			return &admissionreg.ValidatingWebhookConfiguration{
 				ObjectMeta: metav1.ObjectMeta{
-					Name: namePrefix + "-hook",
+					Name: name,
```

Comment:
Is it not possible to use `GenerateName` here?

Reply:
Did not know about this field. Thanks!